CHANGES.md

```diff
@@ -0,0 +1,125 @@
+## DC/OS 1.11.3
+
+
+### Notable changes
+
+* Added support for CoreOS 1688.4.0. (DCOS_OSS-2417)
+
+
+### Fixed and improved
+
+
+### Security updates
+
+
+## DC/OS 1.11.2
+
+
+### Notable changes
+
+* Updated to [Mesos 1.5.1-dev](https://github.com/mesosphere/mesos/blob/27d91e1fe46f09b2c74f2dc4efe4f58ae59ae0a8/CHANGELOG).
```

Comment:
Upgrading to a `-dev` release seems pretty weird, but I guess such is life.

Reply:
This appears to be an artifact of our having a different release cadence to Mesos. That makes total sense.

Reply:
We could actually request mesos team (@rukletsov / @karya0 / @greggomann ) to consider providing a tagged URI like (Metronome).

Reply:
I think in the future we should reference commits from the apache/mesos repo and if possible released versions.